openmetadata-service/src/main/java/org/openmetadata/service/clients/pipeline/airflow/AirflowRESTClient.java

```diff
@@ -230,7 +230,7 @@ public Response getServiceStatus() {
   }
 
   @Override
-  public Response testConnection(TestServiceConnection testServiceConnection) {
+  public Response testConnection(TestServiceConnectionRequest testServiceConnection) {
```

Comment:
For now we just align the imports. In a followup PR we'll update from a `testConnection` endpoint, to a `runOperationsWorkflow` one. This can then be developed in each pipeline service to execute the test or any other process